pkgs/top-level/python-packages.nix

```diff
@@ -366,6 +366,14 @@ in {
 
   hdmedians = callPackage ../development/python-modules/hdmedians { };
 
+  hoomd-blue = callPackage ../applications/science/molecular-dynamics/hoomd-blue {
+    inherit python;
+  };
+
+  hoomd-blue-mpi = (self.hoomd-blue.override {
```

Comment:
Let's drop this one. We don't include all possible ways something can be build because of limited CI capacity.

Reply:
Fixed. I have droped hoomd-blue-mpi from attributes.